src/view-data/method.ts

```diff
@@ -113,9 +113,13 @@ function getPathToMethodName(httpVerb: string, path: string): string {
   });
 
   const result = camelCase(segments.join("-"));
-  return `${httpVerb.toLowerCase()}${result[0].toUpperCase()}${result.substring(
-    1
-  )}`;
+  if (result.length > 0) {
+    return `${httpVerb.toLowerCase()}${result[0].toUpperCase()}${result.substring(
+      1
+    )}`;
```

Comment:
For improved readability, can we create descriptively named helper consts, and use those in the formatting?
For instance:

````
const verb = httpVerb.toLowerCase()
if ... {
  const camelCaseName = `${result[0].toUpperCase()}${result.substring(1)};`
  return `${verb}${camelCaseName}`;
}

return `${verb}_root_`;

````

Reply:
Done